app/code/Magento/ConfigurableProduct/view/adminhtml/web/js/components/dynamic-rows-configurable.js

```diff
@@ -219,7 +219,7 @@ define([
             _.each(tmpData, function (row, index) {
                 path = this.dataScope + '.' + this.index + '.' + (this.startIndex + index);
                 row.attributes = $('<i></i>').text(row.attributes).html();
-                row.sku = $('<i></i>').text(row.sku).html();
+                row.sku = row.sku;
```

Comment:
```suggestion
                row.sku = $('<i></i>').html(row.sku).html();
```

Reply:
Hello @swnsma,

If the SKU is like "C&A" with your solutions "$('<i></i>').html('C&A').html();" is become "Best &amp; Test" and that not the correct solution for it.

Thanks